src/Features/CSharp/Portable/CodeCleanup/CSharpCodeCleanupService.cs

```diff
@@ -147,13 +151,19 @@ internal class CSharpCodeCleanupService : ICodeCleanupService
                 var removeUsingsService = document.GetLanguageService<IRemoveUnnecessaryImportsService>();
                 if (removeUsingsService != null)
                 {
-                    document = await removeUsingsService.RemoveUnnecessaryImportsAsync(document, cancellationToken).ConfigureAwait(false);
+                    using (Logger.LogBlock(FunctionId.CodeCleanup_RemoveUnusedImports, cancellationToken))
+                    {
+                        document = await removeUsingsService.RemoveUnnecessaryImportsAsync(document, cancellationToken).ConfigureAwait(false);
+                    }
                 }
             }
 
             if (docOptions.GetOption(CodeCleanupOptions.SortImports))
             {
-                document = await OrganizeImportsService.OrganizeImportsAsync(document, cancellationToken).ConfigureAwait(false);
+                using (Logger.LogBlock(FunctionId.CodeCleanup_SortImports, cancellationToken))
+                {
+                    document = await OrganizeImportsService.OrganizeImportsAsync(document, cancellationToken).ConfigureAwait(false);
+                }
             }
 
             return document;
```

Comment:
using Logger.LogBlock at ApplyCodeFixesAsync got disappeared.

Reply:
Good catch, I added it back.